iOS_SDK/OneSignalDevApp/OneSignalDevApp/AppDelegate.m

```diff
@@ -67,34 +80,43 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Example setter for IAM action click handler using OneSignal public method
     [OneSignal setInAppMessageClickHandler:inAppMessagingActionClickBlock];
 
-    [OneSignal initWithLaunchOptions:launchOptions
-                               appId:[AppDelegate getOneSignalAppId]
-          handleNotificationReceived:notificationReceiverBlock
-            handleNotificationAction:openNotificationHandler
-                            settings:@{kOSSettingsKeyAutoPrompt: @false,
-                                       kOSSettingsKeyInAppLaunchURL: @true}];
+    [OneSignal setAppSettings:@{
+        kOSSettingsKeyAutoPrompt: @false,
+        kOSSettingsKeyInAppLaunchURL: @true
+    }];
+    [OneSignal setAppId:[AppDelegate getOneSignalAppId]];
+    [OneSignal setLaunchOptions:launchOptions];
     
-//    [OneSignal setLocationShared:false];
+    [OneSignal setLocationShared:false];
     
     [OneSignal sendTag:@"someKey1122" value:@"03222017"];
 
     [OneSignal addPermissionObserver:self];
     [OneSignal addSubscriptionObserver:self];
     [OneSignal addEmailSubscriptionObserver:self];
     
-    [OneSignal pauseInAppMessages:false];
-
+    [OneSignal pauseInAppMessages:true];
+    
+    [OneSignal setNotificationWillShowInForegroundHandler:notificationReceiverBlock];
+    [OneSignal setNotificationOpenedHandler:openNotificationHandler];
+    
+    //UNUserNotificationCenter.currentNotificationCenter.delegate = _notificationDelegate;
+    
     NSLog(@"UNUserNotificationCenter.delegate: %@", UNUserNotificationCenter.currentNotificationCenter.delegate);
     
     return YES;
 }
 
-#define ONESIGNAL_APP_ID_KEY_FOR_TESTING @"ONESIGNAL_APP_ID_KEY_FOR_TESTING"
+#define ONESIGNAL_APP_ID_KEY_FOR_TESTING @"77e32082-ea27-42e3-a898-c72e141824ef"
 
 + (NSString*)getOneSignalAppId {
+    NSString* newAppId = @"77e32082-ea27-42e3-a898-c72e141824ef";
     NSString* onesignalAppId = [[NSUserDefaults standardUserDefaults] objectForKey:ONESIGNAL_APP_ID_KEY_FOR_TESTING];
-    if (!onesignalAppId)
-        onesignalAppId = @"0ba9731b-33bd-43f4-8b59-61172e27447d";
```

Comment:
I know I accidentally do this as well, but lets try our best to keep this app_id instead of our own internal testing ones. Helps keep the env to ourselves haha

`0ba9731b-33bd-43f4-8b59-61172e27447d`

Reply:
fixed!